lib/scripts.js

```diff
@@ -230,11 +238,57 @@ var scripts = {
         }
       });
   },
-  moveToCompleted: function(job, removeOnComplete){
+  _moveToCompleted: function(job, removeOnComplete){
```

Comment:
I think `Job.prototype.move()` should be removed to, which would allow us to remove the `move` script (which appears to be redundant).